src/middleware/changeURL.js

```diff
@@ -64,6 +64,10 @@ export const changeURLMiddleware = (store) => (next) => (action) => {
       query.r = action.data === state.controls.defaults.geoResolution ? undefined : action.data;
       break;
     }
+    case types.CHANGE_LANGUAGE: {
```

Comment:
In order to allow more time for discussion on the interaction between language support in nextstrain.org & auspice, please comment out this code & leave a comment explaining that it is to be reintroduced after further discussions. I'm 95% sure what you've done here is the right direction but don't want to introduce a new query that we then remove in a week or two.